x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_details.test.tsx

```diff
@@ -463,6 +465,74 @@ describe('disable button', () => {
     handler!({} as React.FormEvent);
     expect(enableAlert).toHaveBeenCalledTimes(1);
   });
+
+  it('should bring back error banner after re-enable if previously dismissed', async () => {
```

Comment:
Is it expected that I will see the error banner without refreshing the page after reenabling? I see the error banner disappear after disabling but after enabling, I have to refresh the page to see the error banner again

![Apr-26-2021 08-23-42](https://user-images.githubusercontent.com/13104637/116081893-cb25e500-a668-11eb-909c-a38f830225e6.gif)

Reply:
>Is it expected that I will see the error banner without refreshing the page after reenabling? I see the error banner disappear after disabling but after enabling, I have to refresh the page to see the error banner again

After re-enabling the rule, it would have a `pending` status which I believe explains why the banner doesn't re-appear until you refresh the page (or click the refresh button). Because by then, the rule had a chance to run once and update the execution status. Does this experience seem strange?

Reply:
Ah I see. I think that's fine. The test name made me think the error banner should reappear right away but I think it's ok if it doesn't.

Reply:
Ah, I see what you mean. I fixed the test name in this commit: https://github.com/elastic/kibana/pull/98135/commits/624ae75701b90bca54127aca0a6ce980a2047785.